companies/exotargets.json

```diff
@@ -6,10 +6,10 @@
     "name": "eXotargets Data Network GmbH",
     "address": "Hafenstr. 3\n60327 Frankfurt am Main\nGermany",
     "phone": "+49 69 29921932",
-    "email": "dsb-exo@ibs-data-protection.de",
+    "email": "datenschutzanfrage@exotargets.de",
     "web": "https://www.exotargets.com/",
     "sources": [
-        "https://www.exotargets.com/legal/",
+        "https://www.exotargets.com/datenschutz/",
```

Comment:
`/datenschutz`  is a "page not found"/404 for me in both Firefox and Chrome. I think you wanted to refer to the `.de` page: https://www.exotargets.de/datenschutz/

`/legal` seems to contain the same content, but in English? :D

Reply:
Yes, makes sense. I simply applied `/legal/` to the `.de` domain and that was not available, but since this is still working for the `.com` domain, I will revert my change. Thank you for the suggestion.